presto-orc/src/main/java/com/facebook/presto/orc/reader/SliceDictionaryBatchStreamReader.java

```diff
@@ -162,8 +162,8 @@ public Block readBlock()
                 // The only valid case for dataStream is null when data has nulls is that all values are nulls.
                 // In that case the only element in the dictionaryBlock is null and the ids in idsVector should
                 // be all 0's, so we don't need to update idVector again.
-                int nullValues = presentStream.getUnsetBits(nextBatchSize);
-                if (nullValues != nextBatchSize) {
+                int notNullValues = presentStream.countBitsSet(nextBatchSize);
```

Comment:
This change doesn't seem to be related?

Reply:
@rongrong this change was in the original commit "Improve ORC boolean reader". That commit removed public int getUnsetBits(int batchSize) from BooleanInputStream. Since this is cherry-pick, should we maintain the original commit structure?

Reply:
I removed this change from  "Improve ORC boolean reader" since the SliceDictionaryBatchStreamReader improvement is not included in this PR. But please note that `26cede88e8 Improve ORC LongReader` contains changes other than the LongReaders, e.g. Map, List and Slice readers. It was because `default void nextIntVector(int items, int[] vector, int vectorOffset, boolean[] isNull)' was removed from `LongInputStream` in this commit.